pkg/kubelet/kubelet.go

```diff
@@ -2803,11 +2803,28 @@ func (kl *Kubelet) handlePodResourcesResize(pod *v1.Pod) *v1.Pod {
 		return pod
 	}
 
+	// If the pod cannot be resized, use the old resource spec during the rest of SyncPod()
+	// for a case where a container is restarted.
+	// Extract the old spec from the latest container status.
+	copyPodResourceToSpec := func(podCopy *v1.Pod) {
```

Comment:
do you think a comment like
```
It may be tempting to use the container.Resources.Requests above, but those are static and multiple successive resizes can only be found from the ContainerStatuses
```
would be helpful? I have to think about it a bit to figure out why we didn't use that

Reply:
Thank you for your feedback. `container.Resources.Requests` should not be used because the values may not be accepted in this issue. I will add more comments to elaborate the issue.